app/utils/data.ts

```
@@ -0,0 +1,314 @@
+        func.apply(context, args);
+      }
+    }, wait);
+  };
+};
+
+/** Alternative of `lodash.shuffle`. It shuffles array's items keeping their references */
+export const shuffle = <T>(arr: T[]): T[] => {
+  const copyArray = arr.slice();
+  let currentIndex = copyArray.length,
+    randomIndex,
+    temporaryValue;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)